ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -1,189 +1,333 @@
+                                <li><a href="#">Encodings</a></li>
+                                <li><a href="#">Tools</a></li>
+                                <li class="divider"></li>
+                                <li><a href="#">My Events</a></li>
+                                @if (Event != null)
+                                {
+                                    <li><a href="#">Event Properties <i>(if admin)</i></a></li>
+                                }
+                                <li><a href="#">Events Dashboard <i>(if admin)</i></a></li>
+                                <li class="divider"></li>
+                                <li><a asp-page="/About">About</a></li>
+                                <li><a asp-page="/Contact">Contact</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <ul class="nav navbar-nav navbar-right">
+                        <li><a href="#"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                        <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login/out</a></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    }
+        <!--CURRENT NAV BAR END-->
+        <!--ADMIN NAV BAR-->
+        @if (Event != null && EventRole != null && EventRole == ModelBases.EventRole.admin)
+        {
+            <nav class="navbar navbar-inverse navbar-fixed-top admin-menu">
+                <div class="container">
+                    <div class="navbar-header">
+                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                            <span class="sr-only">Toggle navigation</span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                        </button>
+                        <a asp-page="/." class="navbar-brand" data-toggle="dropdown">
+                            Admin
+                        </a>
+                        <!-- Get this to work & add it to the other header bars
+                        <ul class="dropdown-menu">
+                            <li><a asp-page="/." asp-route-eventRole="@ModelBases.EventRole.admin">Admin</a></li>
+                            <li><a asp-page="/." asp-route-eventRole="@ModelBases.EventRole.author">Author</a></li>
+                            <li><a asp-page="/." asp-route-eventRole="@ModelBases.EventRole.play">Player</a></li>
+                        </ul>
+                        -->
+                    </div>
+                    <div class="navbar-collapse collapse">
+                        <ul class="nav navbar-nav">
+                            <li><a asp-page="/Events/Details" asp-route-id="@Event.ID" asp-route-eventRole="@ModelBases.EventRole.admin">Event</a></li>

```

Comment:
Does relative navigation work without having to add in the two route parameters?

Reply:
Yes - relative navigation works since the parameters show up before that point in the URL